stories/accordion/subcomponentAccordions.tsx

```diff
@@ -1,19 +1,21 @@
 import React from 'react';
 import { css } from '@emotion/core';
-import { Accordion } from 'src/shared-components';
 
-class DefaultAccordion extends React.Component {
-  state = {
+import { Accordion } from '../../src/shared-components';
```

Comment:
You can get these imports by adding the following to the `stories/tsconfig.json`:

```
"baseUrl": "../",
"rootDirs": ["src"],
...

Reply:
I played a little with these values now each project `src` and `stories` have their own configs and was able to refactor all imports to start from root e.g. `src/...` or `docs/...`
As a side note I also had to add a config to Jest to indicate the root as `./` for some reason it doesnt work without it